### ISSUE TITLE
Fedora 27 netinstall iso

### ISSUE THREAD

**Issue by fbellet** [livecd-tools/livecd-tools#93]
Fedora 27 (workstation) netinstall iso file contains a command line string "inst.stage2=hd:LABEL=Fedora-WS-dvd-x86_64-27" that certainly needs to be tweaked with the $TGTLABEL value instead. This image is different from the live iso files, containing the command line string "root=live:CDLABEL=Fedora-WS-Live-27-1-6" instead.

**Comment by Conan-Kudo** [comment]
@fbellet I'm confused. Which aspect of livecd-tools are you having an issue with?

**Comment by fbellet** [comment]
Sorry if my initial message wasn't clear. I'm having an issue with livecd-iso-to-disk, that works when I put a Fedora 27 live iso image on my usb stick, but fails with the "netinst" iso image. The difference between these two iso images is in the command line.

So a patch like this one makes livecd-iso-to-disk also work with netinst iso images:
```
--- a/tools/livecd-iso-to-disk.sh
+++ b/tools/livecd-iso-to-disk.sh
@@ -1897,6 +1897,7 @@ fi
 echo "Updating boot config files."
 # adjust label and fstype
 sed -i -r "s/\<root=[^ ]*/root=live:$TGTLABEL/g
+        s/inst.stage2=hd:LABEL=[^ ]*/inst.stage2=hd:$TGTLABEL/g
         s/\<rootfstype=[^ ]*\>/rootfstype=$TGTFS/" $BOOTCONFIG $BOOTCONFIG_EFI
 if [[ -n $kernelargs ]]; then
     sed -i -r "s;=initrd.?\.img\>;& ${kernelargs} ;


```

**Comment by FGrose** [comment]
The suggested patch is contained in pull request #109.

**Comment by fbellet** [comment]
Thank you for this inclusion!

**Comment by Conan-Kudo** [comment]
Fixed in livecd-tools v26.0.
